bid.proto

```diff
@@ -15,25 +15,23 @@ message Bid {
     JubJubCompressed commitment = 5;
     BlsScalar elegibility_ts = 6;
     BlsScalar expiration_ts = 7;
-    uint64 bid_tree_storage_index = 8;
+    fixed64 bid_tree_storage_index = 8;
 }
 
 // Used to Request the creation of a Bid
 message BidTransactionRequest {
     BlsScalar k = 1;
-    uint64 value = 2;
+    fixed64 value = 2;
     JubJubCompressed secret = 3;
     StealthAddress pk_r = 4;
     BlsScalar seed = 5;
     fixed64 latest_consensus_round = 6;
     fixed64 latest_consensus_step = 7;
-    //rusk.Transaction tx = 8;
 }
 
 message BidTransaction {
-    Bid bid = 1;
-    BlsScalar bid_hash = 2;
-    // rusk.Transaction tx = 3;
+    fixed64 bid_tree_storage_index = 1;
```

Comment:
It's ok to remove the Bid and the hash of the bid for now. But we need to keep an eye on this since Idk if it will be needed for something at some point by the node.

Reply:
Yep, agreed. I dont foresee needing to use it, but if we do I can always change this.